src/tests/test_sexual_behaviour.py

```diff
@@ -33,6 +35,9 @@ def check_num_partners(row):
     sex = row["sex"]
     group = row["sex_behaviour"]
     n = row["num_partners"]
+    age = row["age"]
+    if age <= 15:
+        return n == 0
```

Comment:
I would maybe add a comment here to explain why this is needed, for future readers.